ansible_base/resource_registry/tasks/sync.py

```diff
@@ -317,8 +316,9 @@ def _cleanup_orphans(self, resource_type, manifest_list):
         if self.deleted_count:
             self.write(f"Deleting {self.deleted_count} orphaned resources")
             for orphan in resources_to_cleanup:
-                # If it was created in the latest X seconds, ignore it.
-                if orphan.content_object.created >= timezone.now() - timedelta(seconds=self.retain_seconds):
+                # Skip items that owned by this service, but are partially migrated
+                if orphan.service_id == service_id() and orphan.is_partially_migrated == True:
```

Comment:
case: is_partially_migrated == False
- Can items owned by a service, but hasn't been migrated to Gateway, be deleted? 
- Do gateway items have is_partially_migrated field? If yes, I assume is_partially_migrated in gateway is always False. This means that we always delete Gateway items if they're not in the manifest?

The old code had a check to avoid deleting newly created items, that's why I asked.

Reply:
I'm removing the time check because Hub doesn't have a created field for users :/

We do have `is_partially_migrated` in gateway, but it's not really used for anything.